### PR TITLE
Move media picker files into ui package

### PR DESCRIPTION
**Pull request by planarvoid**
As discussed, this PR moves the Media picker classes from `ui.photopicker.mediapicker` package to `ui.mediapicker`. The reason is to make it easier to remove the deprecated classes once the work on the project is done.

To test:
- there is nothing to test here, this was an automatic conversion

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -577,7 +577,7 @@
             android:theme="@style/WordPress.NoActionBar" />
 
         <activity
-            android:name=".ui.photopicker.mediapicker.MediaPickerActivity"
+            android:name="org.wordpress.android.ui.mediapicker.MediaPickerActivity"
```
Comment (develric) [inline WordPress/src/main/AndroidManifest.xml]: I would consider to remove the base package `org.wordpress.android` for consistency with the other entries. wdyt?

**Review by develric — COMMENTED** [review]
Hi @planarvoid , smoke tested this (and seems ok to me) and left a minor comment. Let me know 🙇‍♂️

File: WordPress/src/main/AndroidManifest.xml (hunk 1 above)
Comment (planarvoid) [reply WordPress/src/main/AndroidManifest.xml]: sure, this was added automatically by AS

**Comment by planarvoid** [comment]
thanks for the review @develric , I've updated the package name in AndroidManifest

**Review by develric — APPROVED** [review]
Thank for the follow up @planarvoid ; LGTM 👍 !

**Pull request merged**